wfe/wfe.go

```diff
@@ -1489,22 +1498,59 @@ func (wfe *WebFrontEndImpl) FinalizeOrder(
 		return
 	}
 
+	// spliting order identifiers per types
+	var orderDNSs []string
+	var orderIPs []net.IP
+	for _, ident := range orderIdentifiers {
+		switch ident.Type {
+		case acme.IdentifierDNS:
+			orderDNSs = append(orderDNSs, ident.Value)
+		case acme.IdentifierIP:
+			orderIPs = append(orderIPs, net.ParseIP(ident.Value))
+		default:
+			wfe.sendError(acme.UnauthorizedProblem(
+				fmt.Sprintf("Order includes illegal ident type %s", ident.Type)), response)
```

Comment:
I try to avoid "illegal" since its somewhat vague.
```suggestion
				fmt.Sprintf("Order includes unknown identifier type %s", ident.Type)), response)
```